hvac/v1/__init__.py

```diff
@@ -271,12 +277,15 @@ def list_policies(self):
         """
         return self._get('/v1/sys/policy').json()['policies']
 
-    def get_policy(self, name):
+    def get_policy(self, name, parse=False):
         """
         GET /sys/policy/<name>
         """
         try:
-            return self._get('/v1/sys/policy/{0}'.format(name)).json()['rules']
+            policy = self._get('/v1/sys/policy/{0}'.format(name)).json()['rules']
+            if parse and has_parser:
+                policy = hcl.loads(policy)
```

Comment:
If pyhcl isn't available but `parse=True`, I feel like this should raise an exception.